### PR TITLE
Fixup in kubelet.go: Package short names should be [a-z0-9]

### PR DESCRIPTION
**Pull request by mtaufen**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32137)

### CODE REVIEW

**Comment by mtaufen** [comment]
Suggestions for a better name are welcome. `componentconfigv1a1` is a bit long, but might work. `componentconfigv1alpha1` feels too long to me. Maybe `cmpcfgv1a1`?

File: pkg/kubelet/kubelet.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/api/validation"
 	"k8s.io/kubernetes/pkg/apis/componentconfig"
-	kubeExternal "k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
+	kubeexternal "k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1"
```
Comment (smarterclayton) [inline pkg/kubelet/kubelet.go]: componentconfigv1alpha1 isn't the end of the world - it's accurate, and probably easier to read than the shortened form due to lack of vowels.

Comment (mtaufen) [inline pkg/kubelet/kubelet.go]: I can agree with that. Clarity > wordiness.

**Comment by mtaufen** [comment]
Also updated `cmd/kubelet/app/server.go` to use the same name for this package.

**Comment by mtaufen** [comment]
@smarterclayton PTAL

**Comment by mtaufen** [comment]
@vishh PTAL

**Pull request merged**